.github/workflows/release-artifacts.yml

```diff
@@ -40,7 +43,10 @@ jobs:
       - name: Install dependencies
         run: |
             brew update-reset
-            brew install gcc cmake protobuf-c
+            brew install gcc cmake protobuf-c rustup
+            rustup default stable
```

Comment:
I'm not 100% sure that just doing `rustup default stable` is enough.

The logs of the dry run you made seem, to me, as if Rust was just updated in the default image since the previous run of the action:

```
...

Warning: rustup-init 1.26.0_1 is already installed and up-to-date.
To reinstall 1.26.0_1, run:
  brew reinstall rustup-init

...

info: using existing install for 'stable-x86_64-apple-darwin'
info: default toolchain set to 'stable-x86_64-apple-darwin'

  stable-x86_64-apple-darwin unchanged - rustc 1.74.0 (79e9716c9 2023-11-13)
```

Seems like it was a no-op.

Reply:
`rustup default stable` does also trigger an update. But I'll test this once more to be 100% sure.

Reply:
Ok, thanks for the clarification.

Reply:
Yes, it does.

The first part of your snippet is likely because `rustup` is already installed through brew. So I can probably skip that step. I'll quickly see what happens without it.

Reply:
Correct. We don't have to install `rustup` again, and removing it also removes the above warning ([job log](https://github.com/qdrant/qdrant/actions/runs/7166924508/job/19511952023#step:2:141)):

```bash
info: using existing install for 'stable-x86_64-apple-darwin'
info: default toolchain set to 'stable-x86_64-apple-darwin'

  stable-x86_64-apple-darwin unchanged - rustc 1.74.0 (79e9716c9 2023-11-13)

Default host: x86_64-apple-darwin
rustup home:  /Users/runner/.rustup

stable-x86_64-apple-darwin (default)
rustc 1.74.0 (79e9716c9 2023-11-13)
cargo 1.74.0 (ecb9851af 2023-10-18)
release: 1.74.0
commit-hash: ecb9851afd3095e988daaa35a48bc7f3cb748e04
commit-date: 2023-10-18
host: x86_64-apple-darwin
libgit2: 1.7.1 (sys:0.18.0 vendored)
libcurl: 8.1.2 (sys:0.4.68+curl-8.4.0 system ssl:(SecureTransport) LibreSSL/3.3.6)
ssl: OpenSSL 1.1.1u  30 May 2023
os: Mac OS 12.7.1 [64-bit]
```